DataProducts/inc/Geom.hh

```diff
@@ -0,0 +1,24 @@
+#ifndef RecoDataProducts_Geom_hh
+#define RecoDataProducts_Geom_hh
+// typedef for cartesian vector used in reconstruction
+/// root
+#include "Math/Vector3D.h"
+#include "Math/Vector4D.h"
+#include "CLHEP/Vector/LorentzVector.h"
+#include "CLHEP/Vector/ThreeVector.h"
+#include <string>
```

Comment:
I don't believe that string is used by this class.  So the include can be removed